Hazel/src/Hazel/Scene/Scene.cpp

```diff
@@ -124,11 +124,11 @@ namespace Hazel {
 		auto view = m_Registry.view<CameraComponent>();
 		for (auto entity : view)
 		{
-			auto& cameraComponent = view.get<CameraComponent>(entity);
+			auto cameraComponent = view.get<CameraComponent>(entity);
```

Comment:
Why did you remove the reference? If you hover over the `get` function, you'll see that it returns a CameraComponent&
```suggestion
			auto& cameraComponent = view.get<CameraComponent>(entity);
```

Reply:
you should leave that `&` there.
removing `&` is when getting multiple components
```c++
auto& cameraComponent = view.get<CameraComponent>(entity); // Correct
auto cameraComponent = view.get<CameraComponent>(entity); // Error

auto& [cam, trans] = (view / group).get<CameraComponent, TransFormComponent>(entity); // Error (for some reason it works for MSVC but I get compile errors when compiling on Linux with gcc / g++
auto [cam, trans]= (view / group).get<CameraComponent, TransFormComponent>(entity); // Correct
```
```suggestion
			auto& cameraComponent = view.get<CameraComponent>(entity);
```

Reply:
With GCC it gives you an error when using a reference

Reply:
That's pretty weird because for it works perfectly fine with gcc. Could you maybe share you error?